content/en/blogs/Lighthouse-a-webhook-handler-for-k8s-devops.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Lighthouse: A Webhook Handler for Kubernetes DevOps'
+keywords: Kubernetes, KubeSphere, DevOps, Lighthouse, ChatOPs
+description: A brief introduction to Lighthouse.
+tag: 'KubeSphere, Kubernetes, DevOps, Lighthouse, ChatOPs'
+createTime: '2021-04-25'
+author: 'Shaowen Chen, Serena'
+snapshot: '/images/blogs/en/lighthouse-a-webhook-handler-for-k8s-devops/lighthouse.png'
+---
+
+In this blog, I will introduce a lightweight ChatOps tool named [Lighthouse](https://github.com/jenkins-x/lighthouse). As a derivation of [Prow](https://github.com/kubernetes/test-infra/tree/master/prow), a GitHub-specific CI/CD system based on Kubernetes, Lighthouse supports multiple Git providers, making it way more convenient for contributors to cooperate.
+
+## What Is Lighthouse
+
+Lighthouse is a ChatOps based lightweight webhook handler that can trigger pipelines and tasks on pipeline engines like Jenkins, Jenkins X, and Tekton through webhooks from Git repositories. Besides, it also supports multiple Git providers such as GitHub, GitHub Enterprise, BitBucket Server, and GitLab.
+
+## Lighthouse Vs Prow: Which Is Better?
+
+Lighthouse is originally derived from Prow. Currently, Lighthouse supports the common Prow plugins as it reuses the Prow plugin source code, allowing it to handle webhooks pushed from branches to trigger specific pipelines. Similar to Prow, Lighthouse also uses `config.yaml` and `plugins.yaml` for configuration, so that it is easier to migrate from Prow to Lighthouse and vice versa.
+
+The differences between these two tools lie in the following two aspects:
+
+- Lighthouse uses `jenkins-x/go-scm`, which supports more Git providers, not just limited to GitHub.
+- Lighthouse uses its own `LighthouseJob` CRD rather than a `ProwJob` CRD.
+
+## Prerequisites
+
+- You have a Kubernetes cluster with [Helm](https://helm.sh/docs/intro/install/) (v2 or v3) installed.
```

Comment:
When you write an article, be more specific about the prerequisites. Change this sentence to:

To install Lighthouse, you need to have a Kubernetes cluster with [Helm](https://helm.sh/docs/intro/install/) (v2 or v3) installed.